frontends/web/src/containers/SubmitInterface.js

```diff
@@ -74,7 +75,12 @@ class SubmitInterface extends React.Component {
     if (!this.context.api.loggedIn()) {
       this.props.history.push(
         "/login?&src=" +
```

Comment:
A quick thing — I think this same logic might need to be used in `UpdateModelInfoInterface.js` ( here is  a [code pointer](https://github.com/facebookresearch/dynabench/blob/main/frontends/web/src/containers/UpdateModelInfoInterface.js#L26) ) because with this PR I don't think the route `tasks/<taskCode>/submit` exists`

Reply:
Nice catch!